cupy/testing/_helper.py

```diff
@@ -1,37 +1,42 @@
+from __future__ import annotations
+
 import contextlib
 import inspect
-from typing import Callable
 import unittest
-from unittest import mock
 import warnings
+from collections.abc import Callable
+from importlib import metadata
+from unittest import mock
 
 import numpy
+from packaging.requirements import Requirement
```

Comment:
[`cupy.testing`](https://docs.cupy.dev/en/latest/reference/testing.html) is a public API (counter part of [numpy.testing](https://numpy.org/doc/stable/reference/routines.testing.html), and would like to avoid introducing new dependency here. How about importing `packaging` within `installed`?

Reply:
```suggestion
```